utils/wasm-instrument/src/lib.rs

```diff
@@ -40,6 +40,7 @@ pub mod syscalls;
 
 pub const GLOBAL_NAME_GAS: &str = "gear_gas";
 pub const GLOBAL_NAME_ALLOWANCE: &str = "gear_allowance";
+pub const GLOBAL_NAME_STATUS: &str = "gear_status";
```

Comment:
If I'm not mistaken it doesn't get used yet. Remove please.

Reply:
Delete currently

Reply:
removed